pages/common/tailscale-file.md

```diff
@@ -0,0 +1,13 @@
+# tailscale file
+
+> Send files across connected devices on a Tailscale network.
+> It currently does not support sending files to devices owned by other users even on the same Tailscale network.
+> More information: <https://tailscale.com/kb/1106/taildrop/>.
+
+- Send a file to another device on the Tailscale network:
+
+`sudo tailscale file cp {{/path/to/file}} {{hostname_or_ip}}:`
+
+- Receive a file that was sent to the device:
```

Comment:
The file was already received, the sender could be disconnected from the network. This is move the file from a temporal directory to a specific one.
```suggestion
- Store files that was sent to the current node into a specific directory:
```